contracts/MerkleLib.sol

```diff
@@ -105,4 +106,23 @@ library MerkleLib {
         uint256 claimedBitIndex = index % 256;
         claimedBitMap[claimedWordIndex] = claimedBitMap[claimedWordIndex] | (1 << claimedBitIndex);
     }
+
+    /**
+     * @notice Tests whether a claim is contained within a 1D claimedBitMap mapping.
+     * @param claimedBitMap a simple uint256 value, encoding a 1D bitmap.
+     * @param index the index to check in the bitmap.
+     \* @return bool indicating if the index within the claimedBitMap has been marked as claimed.
+     */
+    function isClaimed1D(uint256 claimedBitMap, uint256 index) public pure returns (bool) {
```

Comment:
these are the new merkleLib methods I added. v simple and adapted from the original ones to take in a `uint256 claimedBitMap` rather than `mapping(uint256 => uint256) storage claimedBitMap`